### ISSUE TITLE
training progress 

### ISSUE THREAD

**Issue by NoahBU** [xuannianz/EfficientDet#55]
hi, thanks for sharing. I have a question about training progress.
I train the network on one 1080ti,using:
python3 train.py --snapshot imagenet --phi 1 --gpu 0 --random-transform --compute-val-loss --freeze-backbone --batch-size 4 --epochs 5 --steps 10 pascal /home/***/data/VOCdevkit/VOC2012 

After fist epoch ended, it appears:
Running network:  20% (1172 of 5823) | #####                        Elapsed Time: ......

Is this part doing Network searching progress ? if so, the first epch is training on which network ?
